packages/rancher-windows-exporter/charts/scripts/copy-binary.ps1

```diff
@@ -0,0 +1,40 @@
+$ErrorActionPreference = 'Stop'
```

Comment:
Copied straight from https://github.com/rancher/system-charts/blob/3e8ab13429caede329976f7f556b3cac8b0421cd/charts/rancher-monitoring/v0.2.1/charts/exporter-node-windows/templates/configmap.yaml#L11, this gets placed in a ConfigMap just like before.

Just using helm templating here to make it easier to maintain